src/db.ts

```diff
@@ -1,3 +1,5 @@
 import Knex from "knex";
+// @ts-ignore
```

Comment:
not a critical for this case as we use this library only once

but good to add TS types where it's possible